gcloud-java-dns/README.md

```diff
@@ -160,6 +160,7 @@ IP address 12.13.14.15. Start by adding
 
 ```java
 import com.google.gcloud.dns.ChangeRequest;
+import com.google.gcloud.dns.ChangeRequestInfo;
```

Comment:
I believe the `ChangeRequest` import is no longer needed. Might be wrong though.